inference-engine/tests/functional/plugin/shared/src/single_layer_tests/gather_tree.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/precision_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+
+#include "single_layer_tests/gather_tree.hpp"
+
+namespace LayerTestsDefinitions {
+std::string GatherTreeLayerTest::getTestCaseName(const testing::TestParamInfo<GatherTreeParamsTuple> &obj) {
+    std::vector<size_t> inputShape;
+    InferenceEngine::Precision netPrecision;
+    std::string targetName;
+
+    std::tie(inputShape, netPrecision, targetName) = obj.param;
+
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetName;
+    return result.str();
+}
+
+void GatherTreeLayerTest::SetUp() {
+    std::vector<size_t> inputShape;
+    InferenceEngine::Precision netPrecision;
+
+    std::tie(inputShape, netPrecision, targetDevice) = GetParam();
+
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto paramsIn = ngraph::builder::makeParams(ngPrc, { inputShape , inputShape, {inputShape.at(BATCH_SIZE)}, {}});
```

Comment:
looks like we can use parameter and constant for inputs

Reply:
Constants were added in https://github.com/openvinotoolkit/openvino/pull/2006/commits/1b03a2ea6b993237b40334822ee4a8082c30cff5.